### PR TITLE
Ic solutions info theory

### PR DESCRIPTION
**Pull request by patriciomassaro**

### CODE REVIEW

**Comment by patriciomassaro** [comment]
[IC Solutions SL - Sheet 5](https://trello.com/c/nK3cJ1m3/232-ic-solutions-sl-sheet-5)

File: exercises/information-theory/ex_rnw/ic_sol_entropy_binary_classification.Rnw [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+  
+  
+  
+  \item
+  
+  Now $\epsilon$ has a deterministic relation with $x$. Let's see how our results change with this modification.
+  
+  \begin{itemize}
+    \item
+      As $\epsilon$ and $x$ are have a deterministic relation, the uncertainty introduced by the $\psi$ function is 0. The entropy of $\epsilon$ is given only by the entropy of $x$
```
Comment (ludwigbothmann) [inline exercises/information-theory/ex_rnw/ic_sol_entropy_binary_classification.Rnw:52]: delete "are"

**Review by ludwigbothmann — CHANGES_REQUESTED** [review]
Please also add the pdfs in exercise-pdf in the end

File: exercises/information-theory/ex_rnw/ic_sol_entropy_binary_classification.Rnw [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+      H(\epsilon) &= - \sum_{\epsilon \in \Omega_\epsilon}  \P (\epsilon) log (\P(\epsilon)) \\
+      &= - \theta_\epsilon log(\theta_\epsilon) - (1-\theta_\epsilon) log(1-\theta_\epsilon)
+    \end{align*}
+  \lz
+  
+  \item 
+    The conditional entropy quantifies the uncertainty of $y$ if the outcome of $x$ is given. It is defined as the expected value of the entropies of the conditional distributions, averaged over the conditioning random variable.
+    
+    \begin{align*}
+      H(y|x) &= \E_x [H(y|x=x)] = \sum_{x \in \Omega_x } H(y|x=x) \\
```
[the remark this answers is not in the document]
Comment (patriciomassaro) [reply exercises/information-theory/ex_rnw/ic_sol_entropy_binary_classification.Rnw]: Yes, you are right. But I wrote $P(x=x)$ instead, as theta is already defined as the probability that x = 1

**Comment by patriciomassaro** [comment]
- PDFs added
- Solved the typos
- Instead of adding a theta, I added something different after the sum, please take a look at that and let me know your opinion.

**Comment by patriciomassaro** [comment]
- I moved the Entropy exercise to the second IC exercise sheet of Info Theory
- I added the KL divergence exercise to the first IC exercise sheet and uploaded a draft for the solution.

**Review by ludwigbothmann — APPROVED** [review]

**Pull request merged**